### PR TITLE
[Feature Fix] Fixed long profile names being cutoff.  Closes #2100

### LINKED ISSUE #2100: [Staging & Production] Profile page obscured view for really long names
## Steps
1. Change your profile name to a really long name.
2. Go to your profile page.

## Expected 
For long names the entire name would be visible and not obscured.

## Actual
For long names the portion that moves above is half obscured.

screen shot 2015-03-09 at 10 05 05 am

### PR DESCRIPTION
**Pull request by ticklemepierce**
Purpose
-----------
Closes #2100.  Previously, users with long names would have the name be clipped/cut-off at the top.  This PR fixes this issue so that if the name overflows onto another line, everything is still visible.

Changes
------------
There was a class defined in site.css that was being applied to the name.  This was also the name of a class from Bootstrap, meaning both were being applied even though only one was necessary.  Through searching, it appears that the class definition in site.css is outdated and now deprecated.  It has been deleted so that it does not get inadvertently applied to more things.

Side effects
----------------
I could not find anything that had this class applied to it that should be affected by this deletion, however it is certainly possible.  The following is a list of places where that class may be an issue:
* The various "Claim account" screens
* Dashboard
* Merge accounts
* Resend Confirmation email
* Account settings
* Configure add-ons
* Notifications
* Settings/profile information
* Reset password
* Various project elements
   * Contributors
   * Files
   * Forks 
   * Registrations
   * Settings
   * Statistics
* About page
* How OSF works
* About the Reproducibility Project
* Public Activity

### CODE REVIEW

**Comment by lbanner** [comment]
Thanks @ticklemepierce  ... @caneruguz can you review this one? TY

**Comment by samanehsan** [comment]
@lbanner, I just noticed you asked @caneruguz to review this so I unassigned myself from reviewing.

**Comment by caneruguz** [comment]
@ticklemepierce Can you add a screenshot with the user name same length as the original issue so we can see visually how things have changed. 

You probably resolved the original issue but what happens with very long names now?

**Comment by caneruguz** [comment]
You say there was a class, it's best to state that it is '.page-header' so that we don't have to hunt for it. That also is not deprecated as far as I know but yes I don't know why there is an overflow:hidden on it.

**Comment by ticklemepierce** [comment]
@caneruguz I meant deprecated in that regards.  It is a bootstrap class that was then also in site.css.  Based on the commit when it was added in site.css, this definition of it was deprecated, while of course the bootstrap definition is not.

**Comment by ticklemepierce** [comment]
screen shot 2015-06-01 at 4 59 09 pm

**Comment by caneruguz** [comment]
@Saman this looks ready to merge.

File: website/static/css/site.css [hunk 1]
```diff
@@ -1768,10 +1762,17 @@ div.profile-fullname{
     bottom: 0;
 }
 
-span#profileFullname{
-    position: absolute;
-    bottom: 0;
-    margin-bottom: 20px;
+.avatar_div {
```
Comment (sloria) [inline website/static/css/site.css]: As per COSDev, use `lower-dashed-case` for CSS classes.

**Comment by sloria** [comment]
Minor style change and this is good to merge.

**Pull request merged**